docs/release-notes/release-notes-0.7.1.md

```diff
@@ -0,0 +1,117 @@
+
+Upgrading 
+=========
+
+To upgrade:
+
+(If you are upgrading from a version pre-0.6.2 then please see the section "Move user data to home directory" in [the 0.6.2 release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.6.2.md), and follow the instructions there (except the `commitmentlist` file - that can actually be left alone, the previous release notes were wrong on this point).)
+
+(If you are upgrading from a version pre-0.7.0 please read the "Upgrading" section in [the 0.7.0 release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.6.2.md).)
+
+First run the `install.sh` script as mentioned in the README. When prompted to overwrite the directory `jmvenv`, accept.
+
+Note that `./install.sh -?` will show the options for installation.
+
+Notable changes
+===============
+
+### BIP 78 (Payjoin) Receiver
+
+Joinmarket should now be able to send *and* receive payjoins.
+
+For help in usage, see the [documentation](../PAYJOIN.md). There is also a simple video demonstration [here](https://video.autizmo.xyz/videos/watch/7081ae10-dce0-491e-9717-389ccc3aad0d).
+This completes Joinmarket's implementation of [BIP 78 Payjoin](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki). The receiver uses a Tor hidden service, so Tor must be running on the receiver's machine (see the docs for some details on this). There have been some limited compatibility tests with other wallets, but more testing in the field will be helpful (please report on this github repo if you find issues in using).
+Note in particular that the wallet type (native segwit, or p2sh segwit) may be a relevant consideration, and also don't forget that BIP78 is designed to make sure the payment goes through, even if the coinjoin element fails.
+
+Note, the PSBT and python-bitcointx bump commits are a result of testing with Wasabi that revealed a new (last few months addition to BIP174 (PSBT) that allows NONWITNESS_UTXO fields to be included along with WITNESS_UTXO fields to aid hardware wallet security. These commits allow Joinmarket to correctly process PSBTs that choose this option.
+
+`23d0b8f` BIP78 receiver over a Tor hidden service.
+
+`2b898be` Fix finalization check of PSBT Inputs
+
+`2da073b` Bump python-bitcointx to v1.1.1.post0
+
+`9886553` Adds close button to BIP78 receiver dialog
+
+### Transaction broadcast via peers
+
+A long standing issue since Joinmarket-clientserver was created as an alternative implementation: the original version of Joinmarket allowed participants to request transaction broadcast over the message channel, instead of broadcasting it themselves, for further obfuscation. But part of the code required for this was not implemented. This update corrects that, and now both `random-peer` and `not-self` broadcast options (set in `[POLICY]` tx_broadcast` config option), as documented in the config file, are allowed again.
```

Comment:
```suggestion
A long standing issue since Joinmarket-clientserver was created as an alternative implementation: the original version of Joinmarket allowed participants to request transaction broadcast over the message channel, instead of broadcasting it themselves, for further obfuscation. But part of the code required for this was not implemented. This update corrects that, and now both `random-peer` and `not-self` broadcast options (set in `[POLICY]` `tx_broadcast` config option), as documented in the config file, are allowed again.
```
Should this be `tx_broadcast`? Or there is a superfluous `

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/701/commits/01aad72ef0a5413240fa8825ee75833cba994e11